### PR TITLE
synology-image-assistant 1.0.0-43 (new cask)

### PR DESCRIPTION
**Pull request by piijai**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

Please check the uninstall section, it's my first cask, with a .pkg installer file.

### CODE REVIEW

File: Casks/s/synology-image-assistant.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  url "https://global.download.synology.com/download/Utility/SynologyImageAssistant/#{version}/Mac/SynologyImageAssistant-#{version}-#{arch}.pkg"
+  name "Synology Image Assistant"
+  desc "Assistant to generate image previews of formats like HEIC and HEVC formats"
+  homepage "https://www.synology.com/"
+
+  livecheck do
+    url "https://www.synology.com/en-us/releaseNote/SynologyImageAssistant"
+    regex(/>\s*Version:\s*(\d+(?:\.\d+)*-\d+)\s*</i)
+  end
```
Comment (bevanjkay) [inline Casks/s/synology-image-assistant.rb:16]:
```suggestion
  livecheck do
    url "https://utyupdate.synology.com/electron/SynologyImageAssistant/latest-mac.yml"
    strategy :electron_builder
  end

File: Casks/s/synology-image-assistant.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+cask "synology-image-assistant" do
+  arch arm: "arm64", intel: "x64"
+
+  version "1.0.0-43"
+  sha256 arm:   "322ba03618fff941ce4af008de124523cdfc604b1beb4bfaafdce6efbd636da3",
+         intel: "07cf27ff568dc30c3be6ecc74116fc71ed479fd7499d53a7840cb56ef29b055e"
+
+  url "https://global.download.synology.com/download/Utility/SynologyImageAssistant/#{version}/Mac/SynologyImageAssistant-#{version}-#{arch}.pkg"
+  name "Synology Image Assistant"
+  desc "Assistant to generate image previews of formats like HEIC and HEVC formats"
```
Comment (khipp) [inline Casks/s/synology-image-assistant.rb:10]:
```suggestion
  desc "Assistant to generate image previews of formats like HEIC and HEVC"
```

**Review by khipp — APPROVED** [review]
Thank you!

**Comment by bevanjkay** [comment]
@piijai did you mean to close this PR?

**Comment by piijai** [comment]
@bevanjkay Oh, sorry. No, I meant to commit both reviews and have this cask added to the repo. My mistake was, that I have deleted the branch in my fork repo too soon. Shall I reopen it in a new branch?

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**